speak.md

```diff
@@ -5,22 +5,99 @@ permalink: /speak/
+* [Lilly Ryan](<mailto:mentoring@attacus.net>): experienced international speaker, security engineer, former historian, and current Papers Chair for PyCon AU 2018
+* [Richard Jones](): Python Software Foundation Fellow, former PyCon AU organiser, speaker, and engineer
+
+## How to write your proposal
+
+If you have an idea (or don't!) and want to speak, here's a very rough process of what you should do next:
+
+* Brainstorm or mind map to expand upon your ideas or knowledge in search of a general topic
+* Write a paragraph or two, or some bullet points, to outline the core concepts you want to communicate and what people might learn from your talk
+* Get someone you trust to read your notes and tell you what they think they'd learn
+* Ask one of our mentors for help with building up your submission
+* Practice!
+
+This [public speaking repository](https://github.com/vmbrasseur/Public_Speaking), maintained by [VM Brasseur](https://twitter.com/vmbrasseur), has many useful resources to help you polish your proposals and talks.
+
+## Conduct and expectations
+
+All speakers will be expected to have read and adhere to the conference Code of Conduct, which can be found [here](http://2018.pycon-au.org/conduct/). In particular for speakers: slide contents and spoken material should be appropriate for a professional audience including people of many different backgrounds. Sexual language and imagery is not appropriate, and neither are language or imagery that denigrate or demean people based on race, gender, religion, sexual orientation, physical appearance, disability, or body size.
+
+## Anonymity
+
+Our submission process is anonymous. This means our reviewers will not know anything about your identity while assessing your proposal.
+
+Please avoid including identifying information like your name or your pronouns (he/she/they) anywhere in your proposal so that our reviewers can assess your talk by itself.
+
+## Benefits
+
+Proposals which are accepted will receive one free ticket to PyCon Australia 2018. You are welcome to propose a talk with more than one speaker, but please be aware that if it is accepted, only one complimentary ticket will be allocated.
+
+## Financial support
+
+PyCon Australia has a budget for providing financial assistance to those who may need it. Speakers are very welcome to apply, but please note that financial assistance for speakers is not guaranteed.
+
+You can read more about our policy and submit a financial assistance application [here](http://2018.pycon-au.org/assistance).
 
- * Projects that use Python in interesting ways
- * Development of CPython
- * Testing code
- * Professional skills
- * Interesting bits of Python
- * Python on other platforms
- * ...
+## Have a question? Unsure about anything?
 
-Thinking of something else? We want to hear from you!
+If you have questions about the CFP process, you can reach us any time at **<mailto:program@pycon-au.org>**.
```

Comment:
How about adding the Twitter account here as well? I don't know if or to what degree you want to follow it and reply there.

Reply:
We specifically do not want to break proposal anonymity, and have setup this email for this exact purpose.

Reply:
Gotcha. Great idea. Do you want to make that explicit?